app/assets/stylesheets/pano/global/_flexbox.sass

```diff
@@ -0,0 +1,16 @@
+.flex-row
+  display: flex !important
+  flex-direction: row
+
+  .vertically-centered-column
```

Comment:
vertically-centered-column is a little verbose, maybe something like justify-center-column?